### PR TITLE
Versioning change

### PR DESCRIPTION
**Pull request by bbezuk**

### CODE REVIEW

File: src/highrise/models.py [hunk 1]
```diff
@@ -368,9 +368,14 @@ class SessionMetadata:
...
 
     This will be sent once, as the first message when a connection is established.
+    user_id is the bot's user id.
+
+
     """
 
     user_id: str
+    rate_limits: dict[str, tuple[int, float]]
+    sdk_versions: dict[str, str] | None = None
```
Comment (Tinche) [inline src/highrise/models.py:378]: Can this just be a `str | None`? None if no recommendation, and a version string if the user is recommended to upgrade?

File: src/highrise/__main__.py [hunk 2]
```diff
@@ -58,7 +60,11 @@ async def main(bot: BaseBot, room_id: str, api_key: str) -> None:
                 async with ClientSession() as session:
                     async with session.ws_connect(
                         environ.get("HR_WEBAPI_URL", "wss://highrise.game/web/webapi"),
-                        headers={"room-id": room_id, "api-token": api_key},
+                        headers={
+                            "room-id": room_id,
+                            "api-token": api_key,
+                            SDK_NAME: version,
```
Comment (Tinche) [inline src/highrise/__main__.py:66]: I think we should use the `user-agent` header here, it's standardized and fits the use case well.

Like `highrise-bot-sdk/version`.

**Review by Tinche — CHANGES_REQUESTED** [review]

File: src/highrise/models.py (hunk 1 above)
Comment (bbezuk) [reply src/highrise/models.py]: Done

File: src/highrise/__main__.py (hunk 2 above)
Comment (bbezuk) [reply src/highrise/__main__.py]: Changed

**Pull request merged**